crates/bevy_pbr/src/render/mesh_view_types.wgsl

```diff
@@ -109,3 +109,18 @@ struct ClusterOffsetsAndCounts {
     data: array<vec4<u32>, 1024u>,
 };
 #endif
+
+struct ReflectionProbe {
+    // This is stored as the transpose in order to save space in this structure.
+    // It'll be transposed in the `environment_map_light` function.
+    inverse_transpose_transform: mat3x4<f32>,
+    cubemap_index: i32,
+};
+
+struct LightProbes {
+    // This must match `MAX_VIEW_REFLECTION_PROBES` on the Rust side.
+    reflection_probes: array<ReflectionProbe, 8u>,
+    reflection_probe_count: i32,
+    // -1 if no cubemap is associated.
```

Comment:
What is the separate cubemap index for? The view environment map right? Can you clarify the name?

Reply:
I renamed it to `view_cubemap_index`.